projects/1.21/assets/gardens-of-the-dead/gardens_of_the_dead/lang/zh_cn.json

```
@@ -0,0 +1,52 @@
+    "block.gardens_of_the_dead.soul_spore": "灵魂菌索",
+    "block.gardens_of_the_dead.glowing_soul_spore": "发光灵魂菌索",
+    "block.gardens_of_the_dead.soulblight_fungus": "枯魂菌",
+    "block.gardens_of_the_dead.soulblight_sprouts": "枯魂菌簇",
+    "block.gardens_of_the_dead.blistercrown": "疱冠菌",
+    "block.gardens_of_the_dead.tall_blistercrown": "高疱冠菌簇",
+    "block.gardens_of_the_dead.whistlecane": "鸣怨竹",
+    "block.gardens_of_the_dead.soulblight_stem": "枯魂菌柄",
+    "block.gardens_of_the_dead.stripped_soulblight_stem": "去皮枯魂菌柄",
+    "block.gardens_of_the_dead.soulblight_hyphae": "枯魂菌核",
+    "block.gardens_of_the_dead.stripped_soulblight_hyphae": "去皮枯魂菌核",
+    "block.gardens_of_the_dead.blightwart_block": "枯魂疣块",
+    "block.gardens_of_the_dead.potted_soul_spore": "枯魂菌索盆栽",
+    "block.gardens_of_the_dead.potted_glowing_soul_spore": "发光枯魂菌索盆栽",
+    "block.gardens_of_the_dead.potted_soulblight_fungus": "枯魂菌盆栽",
+    "block.gardens_of_the_dead.potted_soulblight_sprouts": "枯魂菌簇盆栽",
+    "block.gardens_of_the_dead.potted_blistercrown": "疱冠菌盆栽",
+    "block.gardens_of_the_dead.potted_whistlecane": "鸣怨竹盆栽",
+    "block.gardens_of_the_dead.soulblight_planks": "枯魂木板",
+    "block.gardens_of_the_dead.soulblight_slab": "枯魂木台阶",
+    "block.gardens_of_the_dead.soulblight_stairs": "枯魂木楼梯",
+    "block.gardens_of_the_dead.soulblight_button": "枯魂木按钮",
+    "block.gardens_of_the_dead.soulblight_pressure_plate": "枯魂木压力板",
+    "block.gardens_of_the_dead.soulblight_fence": "枯魂木栅栏",
+    "block.gardens_of_the_dead.soulblight_fence_gate": "枯魂木栅栏门",
+    "block.gardens_of_the_dead.soulblight_door": "枯魂木门",
+    "block.gardens_of_the_dead.soulblight_trapdoor": "枯魂木活板门",
+    "block.gardens_of_the_dead.soulblight_sign": "枯魂木告示牌",
+    "block.gardens_of_the_dead.soulblight_hanging_sign": "悬挂式枯魂木告示牌",
+    "block.gardens_of_the_dead.whistlecane_block": "鸣怨竹块",
+    "block.gardens_of_the_dead.whistlecane_planks": "鸣怨竹板",
+    "block.gardens_of_the_dead.whistlecane_mosaic": "鸣怨竹马赛克",
+    "block.gardens_of_the_dead.whistlecane_slab": "鸣怨竹台阶",
+    "block.gardens_of_the_dead.whistlecane_stairs": "鸣怨竹楼梯",
+    "block.gardens_of_the_dead.whistlecane_mosaic_slab": "鸣怨竹马赛克台阶",
+    "block.gardens_of_the_dead.whistlecane_mosaic_stairs": "鸣怨竹马赛克楼梯",
+    "block.gardens_of_the_dead.whistlecane_fence": "鸣怨竹栅栏",
+    "block.gardens_of_the_dead.whistlecane_fence_gate": "鸣怨竹栅栏门",
+    "block.gardens_of_the_dead.whistlecane_button": "鸣怨竹按钮",
+    "block.gardens_of_the_dead.whistlecane_pressure_plate": "鸣怨竹压力板",
+    "block.gardens_of_the_dead.whistlecane_door": "鸣怨竹门",
+    "block.gardens_of_the_dead.whistlecane_trapdoor": "鸣怨竹活板门",
+    "block.gardens_of_the_dead.whistlecane_sign": "鸣怨竹告示牌",
+    "block.gardens_of_the_dead.whistlecane_hanging_sign": "悬挂式鸣怨竹告示牌",
+    "gardens_of_the_dead.subtitle.block.whistlecane.whistle": "鸣怨竹：鸣响",
+    "tag.item.gardens_of_the_dead.soulblight_stems": "枯魂菌方块",
+    "tag.item.gardens_of_the_dead.whistlecane_blocks": "鸣怨竹方块"

```

```suggestion
    "tag.item.gardens_of_the_dead.soulblight_stems": "枯魂菌木",
    "tag.item.gardens_of_the_dead.whistlecane_blocks": "鸣怨竹块"
```
和EMI的翻译对齐
虽然tag其实完全没有翻译准则就是了（）